cardano-node/src/Cardano/Node/Types.hs

```diff
@@ -94,21 +93,6 @@ instance FromJSON GenesisFile where
   parseJSON invalid = panic $ "Parsing of GenesisFile failed due to type mismatch. "
                            <> "Encountered: " <> Text.pack (show invalid)
 
--- Node can be run in two modes.
-data ViewMode = LiveView    -- Live mode with TUI
-              | SimpleView  -- Simple mode, just output text.
-              deriving (Eq, Show)
-
-instance FromJSON ViewMode where
```

Comment:
Let's leave this `FromJSON` instance here and put an error message that explains `LiveView`/`SimpleView` has been deprecated and should be removed from the configuration file. Let devops and the wallet team know to expect this failure if their config yaml files haven't been updated accordingly. Let's replace `panic` with `fail` here as well.

Reply:
Done. Now, if the config contains `ViewMode`, I see the message `AesonException "Error in $.ViewMode: ViewMode is deprecated, please remove it from your configuration file."`.